1.8hm.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+//�������� ���������, ��������� ���������� ������� ��������� � �������.
+
+int nullCounter(int *arr, int len)
+{
+	int counter = 0;
+	for (int i = 0; i < len; i++) 
+	{
+		if (arr[i] == 0)
+			counter++;
+	}
+
+	return counter;
+}
+
+int main()
+{
+	const int A[] = {1,2,0,4,5,6,7,8,0};
+
+	printf("Nulls %i", nullCounter(&A,9));

```

Comment:
```suggestion
	printf("Amount of zeros: %i", nullCounter(A, 9));
```
1) Более информативное сообщение должно быть.
2) Зачем передавать ссылку на массив, если нужен сам массив?